site/guide/model-inventory/register-models-in-inventory.qmd

```diff
@@ -17,19 +17,28 @@ To be able to document models using ValidMind's model documentation and validati
 2. Click **Model Inventory** in the left sidebar. 
 <!--- **Need screenshot** --->
    
-3. Click **Register new model**.
+3. Click **Register New Model**.
    
-4. Fill in the required information on the registration form:
+4. Depending on whether or not your organization has custom fields[^2] defined, input the information on the Register New Model panel in either one or two steps:
 
-   - Provide a model name
-   - Select the relevant business unit
-   - Select the relevant model methodology being used
-   - Select the relevant model use case
-   - Provide a purpose statement to explain what the model will be used for
-   - Select the preliminary risk tier for the model
-   - If the model is a vendor model, toggle **Is Vendor Model**[^2] and provide the vendor name.
+   - **Step 1: Basic Model Information** — If no custom fields are defined for your models, simply provide the following information then move on to step #5 below: 
 
-5. Click **Register new model** to create a new entry in the model inventory.
+      - Model name
+      - Type of model, new[^3] or existing
```

Comment:
When I click on **New**, a field to enter the Template pops up:
<img width="781" alt="image" src="https://github.com/user-attachments/assets/14db5667-75dd-4cc0-a236-84bfbd6d9635">

When I click on **Existing**, a field to enter the Status pops up:
<img width="790" alt="image" src="https://github.com/user-attachments/assets/536e8039-166b-406b-847b-e8f920b5311f">

Reply:
This is documented in the footnote ;)

Reply:
So should there be another footnote next to **Existing** that says `[^x]: Existing models require a **Model Status**` or will that get documented somewhere else?